src/com/itacademy/eshop/product/types/author.java

```diff
@@ -0,0 +1,13 @@
+package com.itacademy.eshop.product.types;
+
+public class author {
+    private String author;
```

Comment:
Class name and parameter name should not match. In this case it would make more sense to have a name of the author.